### PR TITLE
Issue #73: Add support for RSS

### PR DESCRIPTION
**Pull request by Vladlis**

### CODE REVIEW

File: releasenotes-builder/src/main/java/com/github/checkstyle/publishers/SourceforgeRssPublisher.java [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+        final OutputStream os = conn.getOutputStream();
+        final String postText = new String(Files.readAllBytes(Paths.get(postFilename)),
+                StandardCharsets.UTF_8);
+        os.write(String.format(POST_TEMPLATE, bearerToken, releaseNumber, postText)
+                .getBytes(StandardCharsets.UTF_8));
+        os.flush();
+
+        final int responseCode = conn.getResponseCode();
+        conn.disconnect();
+        if (responseCode != HttpURLConnection.HTTP_OK && getPostsCount() == postsCountBefore) {
```
Comment (romani) [inline releasenotes-builder/src/main/java/com/github/checkstyle/publishers/SourceforgeRssPublisher.java]: please make make a single line comment on why you do testing for post counts, it is far from obvious.

File: releasenotes-builder/config/suppressions.xml [hunk 2]
```diff
@@ -6,10 +6,13 @@
...
+    <suppress checks="CyclomaticComplexity" files="Main.java" lines="165"/>
     <suppress checks="CyclomaticComplexity" files="CliOptions.java" lines="309"/>
-    <suppress checks="IllegalCatch" files="Main.java" lines="226, 244, 261"/>
+    <suppress checks="IllegalCatch" files="Main.java" lines="230, 248, 265, 282"/>
     <suppress checks="JavadocMethod" files="CliOptions.java"/>
     <suppress checks="ThrowsCount" files="XdocPublisher.java"/>
     <suppress checks="UncommentedMain" files="Main.java"/>
+
+    <!-- Until https://github.com/checkstyle/checkstyle/issues/3590 -->
+    <suppress checks="CustomImportOrder" files="Main.java" lines="33"/>
```
Comment (romani) [inline releasenotes-builder/config/suppressions.xml]: I already agreed to use comments suppressions in code in main project, contributions repo could do the same.
But it is not mandatory in this PR.

File: releasenotes-builder/src/main/java/com/github/checkstyle/publishers/SourceforgeRssPublisher.java (hunk 1 above)
Comment (Vladlis) [inline releasenotes-builder/src/main/java/com/github/checkstyle/publishers/SourceforgeRssPublisher.java]: Done

File: releasenotes-builder/config/suppressions.xml (hunk 2 above)
Comment (Vladlis) [inline releasenotes-builder/config/suppressions.xml]: Done

**Pull request merged**